tests/FakeItEasy.IntegrationTests/StringArgumentConstraints.cs

```diff
@@ -1,19 +1,17 @@
 namespace FakeItEasy.IntegrationTests
 {
     using System;
-    using System.Globalization;
 
     public static class StringArgumentConstraints
     {
         public static string StartsWith(this IArgumentConstraintManager<string> scope, string beginning)
         {
-            return scope.Matches(
-                x => x.StartsWith(beginning, StringComparison.CurrentCulture), string.Format(CultureInfo.CurrentCulture, "Starts with \"{0}\"", beginning));
+            return scope.Matches(x => x.StartsWith(beginning, StringComparison.CurrentCulture), $@"Starts with ""{beginning}""");

```

Comment:
It wasn't CA1305. IIRC, [CA1307](https://msdn.microsoft.com/en-us/library/bb386080.aspx?f=255&MSPPError=-2147217396).